tests/main/interfaces-desktop-host-fonts/task.yaml

```diff
@@ -5,17 +5,24 @@ details: |
     covering the user's spoken language, the host system's fonts are
     bind mounted into the sandbox.
 
-systems: [-ubuntu-core-*]
+systems:
+    - -amazon-linux-2-*
+    - -centos-7-*
+    - -ubuntu-14.04-*
```

Comment:
Why excluding those in addition to UC?

Reply:
Lack of session bus support, those are server systems that only started to support it in centos-8 and derivatives.